site/en/docs/extensions/reference/declarativeNetRequest/index.md

```diff
@@ -161,6 +164,10 @@ method.
 
 ## Implementation details
 
+### web_accessible_resources
+
+When an extension uses declarativeNetRequest APIs to redirect a public resource request to a resource that is not web accessible, it is blocked and will result in a net::ERR_UNSAFE_REDIRECT error. The above holds true even if the resource that is not web accessible is owned by the redirecting extension. To declare resources for use with declarativeNetRequest APIs, the `"web_accessible_resources"` array must be declared and populated in the manifest as documented [here](/docs/extensions/mv3/manifest/web_accessible_resources/).
```

Comment:
```suggestion
When an extension uses declarativeNetRequest APIs to redirect a public resource request to a resource that is not web accessible, it is blocked and will result in a net::ERR_UNSAFE_REDIRECT error. The above holds true even if the resource that is not web accessible is owned by the redirecting extension. To declare resources for use with declarativeNetRequest APIs, populate the [`"web_accessible_resources"`](/docs/extensions/mv3/manifest/web_accessible_resources/) array.
```

Reply:
This does three things:
* Says it with fewer words.
* Changes it from active to passive voice.
* Makes the link sef-descriptive.